9801-logger/9801-logger.jl

```diff
@@ -169,149 +194,169 @@ try
     println("  Run Time     => $RUN_TIME seconds $is_runtime_default")
     println("  Volt Min     => $VOLT_MIN volts $is_volt_min_default")
     println("  Volt Max     => $VOLT_MAX volts $is_volt_max_default")
-    if (!NO_CSV)
+    if (!NO_CSV && !AUTOMATED)
       println("  Output File  => $csv_name $is_name_default")
     end
 
-    global spin_lock
-    iter_count   = 0
-    recordings   = []
-    timestamps   = []
-    times        = []
-    volts        = []
-    volts_bounds = []
-    freqs        = []
-    reals        = []
-    apps         = []
-    pfacs        = []
-    currs        = []
-    apks         = []
+    
+    for test in TESTS
+      println("")
```

Comment:
why is this here?

Reply:
formatting, so all the lines aren't jumbled together

Reply:
I'd move 
```julia
      if (AUTOMATED)
        println("\nRunning $(test) test for ~$(RUN_TIME) seconds")
      end
```
up a few lines and just add the `\n` to the print statement

Reply:
that would make the informative purpose of that line confusing since it would put that statement above the potential sleep statement when it actually executes in the opposite order

Reply:
https://github.com/system76/system76-ee/pull/10/commits/e19d498e9c7a805e7c09a780ee68388b54a98696 is in my opinion worse. There are better ways to do it, but I think it's fine for now to just revert it to the way it was.

Reply:
alrighty, I've reverted it